docs/dev/install.rst

```diff
@@ -0,0 +1,135 @@
+Developer Setup
+===============
+The following describes how to set up an instance of the site on your
+computer for development.
+
+Prerequisites
+-------------
+This guide assumes you have already installed and set up the following:
+
+1. Git_
+2. `Python 3.5`_, `pip 8`_ or higher, and virtualenv_
+3. `Postgres 9.4`_
+4. ``openssl``
+
+These docs assume a Unix-like operating system, although the site should, in
+theory, run on Windows as well. All the example commands given below are
+intended to be run in a terminal.
+
+.. _Git: https://git-scm.com/
+.. _Python 2.7: https://www.python.org/
+.. _pip 8: https://pip.pypa.io/en/stable/
+.. _virtualenv: https://virtualenv.pypa.io/en/latest/
+.. _Postgres 9.4: http://www.postgresql.org/
+
+Installation
+------------
+1. Clone this repository or your fork_:
+
+   .. code-block:: bash
+
+      git clone --recursive https://github.com/mozilla/normandy.git
+      cd normandy
+
+2. Create a virtualenv for Normandy and activate it:
+
+   .. code-block:: bash
+
+      virtualenv venv
+      source ./venv/bin/activate
+
+   .. note::
+
+      Whenever you want to work on Normandy in a new terminal you'll have to
+      re-activate the virtualenv. Read the virtualenv_ documentation to learn
+      more about how virtualenv works.
+
+3. Install the dependencies using pip:
+
+   .. code-block:: bash
+
+      pip install -r requirements.txt
+
+   .. note::
+
+      If you get an error like `pip: error: no such option: --hash`, you are
```

Comment:
Also, all the pip error handling is pretty big. Maybe we should have a separate troubleshooting section for it?